src/app/shell/result/result.component.ts

```diff
@@ -48,6 +49,8 @@ export class ResultComponent implements OnInit, OnDestroy {
 
   isMobileScreen: boolean;
   isHidden: boolean;
+  
+  readonly Workshop = Workshop;
```

Comment:
I would rather change the naming in the component so it will be better to understand. Something like wokrshopDeclination